### ISSUE TITLE
Autosens adjusts BG targets too high

### ISSUE THREAD

**Issue by scottleibrand** [openaps/oref0#152]
Per @jasoncalabrese: With regular target of 115, autosens raises the target too high when detecting lots of sensitivity. Need to limit autosens target adjustments to 1.5x instead of 2x.

**Comment by sulkaharo** [comment]
Highest I can find in our recent logs is around 30% increase, but capping at 50% sounds good. We have definitely have had >50% adjustments in the past and not sure if those helped; I suspect those were caused by bad cannula sites. Which reminds me - I did have this idea that we could factor cannula age into autosens, or at least detect if the ISF dramatically changes after a cannula change event and disregard ISF info from the previous cannula period if this this is the case.

**Comment by jasoncalabrese** [comment]
I'm hoping to have time over the weekend to look into what happened more.

This is what I want to try to explain (mostly a note to myself), I should able to find the exact inputs in my repo:

At one point I saw it adjust 80 to 156

```
Jul 05 11:44:22 indy-e1 openaps-loop:  >>>>>+++++-----+----->>++->>>+++-+-+-+-----------+-------------++++>+>-+-+---++-++---++---+--+---------------++++>>>>++--+-++-++--------------+++++>++->>>>>>++++-------->>>+--->>>>>>>>>>>>++-->>+>>>>>>>>>>>>>>>>>>>>>>>>>++-------++-+++++-----------+--++-++-++++++>++-+----+++---------> 62% of non-meal deviations negative (target 45%-50%)
Jul 05 11:44:22 indy-e1 openaps-loop:  Excess insulin sensitivity detected: ISF adjusted from 80 to 138
```

**Comment by scottleibrand** [comment]
One thing to look at would be how many of those >>> were carb absorption vs. just BG rising for some other reason.

**Comment by scottleibrand** [comment]
Fixed via #152.
